apps/governance.mento.org/instrumentation-client.ts

```diff
@@ -0,0 +1,17 @@
+// This file configures the initialization of Sentry on the client.
+// The added config here will be used whenever a users loads a page in their browser.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: "https://63fde8b2021a9e2bf389484d2bb31da7@o4504211835256832.ingest.us.sentry.io/4509825739849728",
```

Comment:
Let's store the DSN in a `SENTRY_DSN_GOVERNANCE` env var pls so we don't have to duplicate it